sdk/Trace/SpanProcessor/BatchSpanProcessor.php

```diff
@@ -107,7 +107,7 @@ public function onEnd(API\Span $span): void
      */
     public function forceFlush(): void
     {
-        if (null !== $this->exporter && $this->running) {
```

Comment:
there is no need to check here if the processor is running because no span will be recorded in onEnd method if running is set to false.